pkgs/development/compilers/ghc/7.4.2-binary.nix

```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     # GHC has dtrace probes, which causes ld to try to open /usr/lib/libdtrace.dylib
     # during linking
     stdenv.lib.optionalString stdenv.isDarwin ''
-      export NIX_LDFLAGS+=" -no_dtrace_dof"
+      export NIX_LDFLgGS+=" -no_dtrace_dof"
```

Comment:
Huh? That's probably not intentional?

Reply:
I switch between 3 different keyboards a day and sometimes I make mistakes. Will fix